bloomstack_core/patches/v0_0_1/delete_mandatory_resume_attach_property_setter.py

```diff
@@ -0,0 +1,4 @@
+import frappe
+
+def execute():
+	frappe.delete_doc_if_exists("Property Setter", "Job Applicant-resume_attachment-reqd")
```

Comment:
```suggestion
```
add `frappe.delete_doc_if_exists("Property Setter", "Job Applicant-resume_attachment-reqd")` this code in the `bloomstack_core.patches.v0_0_1.removed_mandatory_for_attachment_field` patch